user_events.py

```diff
@@ -803,12 +804,16 @@ def search():
        return jsonify([]), 200
 
 
-@userbp.route('/searchsub', methods=['GET'])
+@userbp.route('/searchsub/<id>', methods=['GET'])
 @role_required('user')
-def searchsub():
+def searchsub(id):
     if request.method == "GET":
        search_term = request.values.get("data")
-       return jsonify(group_subevents_search(search_term, get_admin_group_ids()))
+       results = group_subevents_search(search_term, get_admin_group_ids())
```

Comment:
Hi, @bingzhang,

Functionality-wise, this PR looks fine. I have a question about efficiency though. Would it be more efficient to pass the event ID to the group_subevents_search method and exclude it in the actual MongoDB query using $nin or something similar?